test/IDE/complete_call_arg.swift

```diff
@@ -253,8 +253,8 @@ extension C3 {
 
   func f7(obj: C3) {
     let _ = obj.hasError(#^HASERROR1^#
-    let _ = obj.hasError(a1: #^HASERROR2^#
-    let _ = obj.hasError(a1: IC1, #^HASERROR3^#
+    let _ = obj.hasError(a1: #^HASERROR2?xfail=FIXME^#
+    let _ = obj.hasError(a1: IC1, #^HASERROR3?xfail=FIXME^#
```

Comment:
I think it's a general solver-based completion issue. The lookup the solver does to get the overloads does an isValid() check on the decls it finds and ignores them if they're not. I can change it to let them through but because it changes what was previously an invariant I wanted to leave that as a separate change if you/rintaro are ok with the regression. I'll file a bug if so, or try to fix it here if not.

Reply:
I think the regression is OK if it only applies if the function signature contains an error type. Would be something that we should fix at some point but I think it’s OK to do it in another PR. I’d just like a bug report so we don’t forget about it 😉

Reply:
Awesome, I'm add it in the next push!